### ISSUE TITLE
Have an issue that using src with a placholder.gif prevents loading of images

### ISSUE THREAD

**Issue by sonicpunk** [aFarkas/lazysizes#53]
when using a scr="placeholder.gif" within my image tag, the script does not load my images.  After I remove the src attribute, it works fine.

**Comment by aFarkas** [comment]
Is this happening in all browsers or only in IE? Is "placeholder.gif" a valid URL to an existing image file?

**Comment by aFarkas** [comment]
Ok, this actually directly belongs to your first question.

Your code looks something like this:

```html
<img src="placeholder.gif" data-srcset="img-2.jpg 2x" data-src="img-1.jpg" class="lazyload" data-sizes="auto" />
```

While it is perfectly valid to write it this way. The 1x descriptor fallback is an exception in the spec. And lazySizes can't know that you are using this exception (The only way to detect this would be to implement a srcset parser (which adds about 2kb). Even if lazySizes would know this, this could produce a double download in polyfilled browsers.

Which yields to the following rule: As soon as you have used a) a srcset and b) either the browser supports responsive images or you are using a polyfill, lazySizes will ignore the ``data-src`` attribute.

The correct markup for what you want looks like this:

```html
<img src="placeholder.gif" data-srcset="img-1.jpg 1x, img-2.jpg 2x" class="lazyload" />
```

Note, I have simply added the img-1.jpg to the srcset. I also removed the ``data-sizes="auto"``, because this feature is especially for srcset using the width not the density descriptor.

Which brings us to the last refinement. I would suggest, that you write your markup the following way:

```html
<img src="placeholder.gif" data-srcset="img-1.jpg 980w, img-2.jpg 1960w" class="lazyload" data-sizes="auto" />
```

Reason: With your old code. A Smartphone would have downloaded the large 2x image, although it will be only shown with a layout pixel width of 320-600. With the width descriptor the smartphone will now only load the 980w candidate.
